src/opnsense/service/templates/OPNsense/WebGui/php.ini

```diff
@@ -18,7 +18,7 @@ html_errors = Off
 zlib.output_compression = {% if system.webgui.compression|default('') != "" %}On{%else%}Off{%endif%}
 
 zlib.output_compression_level = {{system.webgui.compression|default('1')}}
-include_path = ".:/usr/local/etc/inc:/usr/local/www:/usr/local/opnsense/mvc:/usr/local/share/pear:/usr/local/opnsense/contrib"
+include_path = ".:/usr/local/share:/usr/local/etc/inc:/usr/local/www:/usr/local/opnsense/mvc:/usr/local/share/pear:/usr/local/opnsense/contrib"
```

Comment:
this would also go to master directly :)

Reply:
#2621